src/app/ApplicationTemplate.Business/KillSwitch/IKillSwitchService.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+
+namespace ApplicationTemplate.Business;
+
+/// <summary>
+/// Service that handles the kill switch.
+/// </summary>
+public interface IKillSwitchService
+{
+	/// <summary>
+	/// Event that is raised when the kill switch is activated.
+	/// </summary>
+	public event EventHandler<KillSwitchEventArgs> KillSwitchActivationChanged;
```

Comment:
I mostly did it because I figured maybe someday we would want to add more things, but, yeah, it's probably better to keep it just an observable until that day comes.

Reply:
and we could also just return an more complex object in the observable if that day comes too